components/panel/_panel.scss

```diff
@@ -10,6 +10,12 @@
       border-color: $color;
     }
   }
+  /* Removes inherited bottom margin to make whitespace inside panel equal */
+  p:last-of-type,
```

Comment:
Won't this result in _any_ of these elements that are the last of their type having no margin? For example, a paragraph with something else underneath will have no `margin-bottom`.

I'm not sure there's an easy solution to this.

Reply:
Just noticed @bennpearson pointed out the same thing above.